### PR TITLE
add check run link

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

add check run link

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: .github/workflows/dapr-test.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
             if (!testCluster) {
               message = "All test clusters are occupied by the other tests. Please try to test later.";
             } else {
-              message = `Found the available test cluster - ${testCluster} for ${targetOs}. Please wait until test is done.`;
+              message = `Found the available test cluster - ${testCluster} for ${targetOs}. Please check the [build status](https://github.com/dapr/dapr/runs/${context.run_id}).`;
```
Comment (wcs1only) [inline .github/workflows/dapr-test.yml:97]: Probably should replace dapr/dapr with GITHUB_REPOSITORY To make this forkable. There's also one for the base github url.

**Review by wcs1only — DISMISSED** [review]

File: .github/workflows/dapr-test.yml (hunk 1 above)
Comment (youngbupark) [reply .github/workflows/dapr-test.yml]: done.

**Review by wcs1only — APPROVED** [review]

**Pull request merged**